scripts/utils/starknet.py

```diff
@@ -115,8 +113,14 @@ async def get_starknet_account(
     address=None,
     private_key=None,
 ) -> Account:
-    address = int(address or ACCOUNT_ADDRESS, 16)
-    key_pair = KeyPair.from_private_key(int(private_key or PRIVATE_KEY, 16))
+    address = address or ACCOUNT_ADDRESS
+    if address is None:
+        raise ValueError("address was not given in arg nor in env variable")
```

Comment:
Maybe we can point to/link to the Deploy section of the readme here, if this error is thrown. 

My rationale being:
- Newer devs might tinker around with the code base and get stuck here
- pointing them to the readme would help if they are new to the concepts of setting environment variables etc etc